website/content/usage/executors/kafka.md

```diff
@@ -22,7 +22,7 @@ Example
 ```json
 "executor": "kafka",
 "executor_config": {
-    "brokerAddress": "localhost:9092",
+    "brokerAddress": "localhost:9092,localhost:9093",
```

Comment:
I have no experience with Kafka, but I'm guessing having two brokers on the same machine is somewhat odd. If so, can you change the example to something more common?

Reply:
Addressed. Put the same example as the official Kafka docs.